src/nvim/search.c

```diff
@@ -3428,9 +3428,9 @@ void find_pattern_in_path(char_u *ptr, Direction dir, size_t len, bool whole, bo
   int old_files;
   int already_searched;
   char_u *file_line;
-  char_u *line;
-  char_u *p;
-  char_u save_char;
+  char *line;
+  char *p;
```

Comment:
Did you forget #20517? You need to cast to `uint8_t` when passing a char in this string to a function.

Reply:
Thanks